eng/pipelines/pullrequest.yml

```diff
@@ -0,0 +1,86 @@
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+    - restapi*
+    - pipelinev3*
```

Comment:
Did you want to keep this one still?

Reply:
Yes, I believe it would be good to keep pipelinev3 around considering package info changes in the very immediate future and the fact that the pullrequest pipeline is going to be one of the ones that's affected the most.